packages/components/avatar/stories/Avatar.stories.tsx

```diff
@@ -25,80 +29,97 @@ export const Overview: Story<AvatarProps> = (args) => {
-        <Avatar {...argsNoSrc} size="medium" variant="app" />
-        <Avatar {...argsNoSrc} size="small" variant="app" />
-        <Avatar {...argsNoSrc} size="tiny" variant="app" />
-      </Flex>
+      <SectionHeading as="h3" marginBottom="spacingS">
+        Loading
+      </SectionHeading>
 
       <Flex
         alignItems="center"
         flexDirection="row"
         gap="spacingS"
         marginBottom="spacingM"
       >
-        <Avatar {...args} size="tiny" />
-        <Avatar {...args} size="small" />
-        <Avatar {...args} size="medium" />
-        <Avatar {...args} size="large" />
-        <Avatar {...args} size="large" variant="app" />
-        <Avatar {...args} size="medium" variant="app" />
-        <Avatar {...args} size="small" variant="app" />
-        <Avatar {...args} size="tiny" variant="app" />
+        <Avatar size="tiny" variant="user" />
+        <Avatar size="small" variant="user" />
+        <Avatar size="medium" variant="user" />
+        <Avatar isLoading size="large" variant="user" />
+        <Avatar size="large" variant="app" />
+        <Avatar size="large" variant="user" />
+        <Avatar size="medium" variant="app" />
+        <Avatar size="small" variant="app" />
+        <Avatar size="tiny" variant="app" />
       </Flex>
 
-      <SectionHeading as="h4">Indicator properties</SectionHeading>
+      <SectionHeading as="h3" marginBottom="spacingS">
+        Indicator properties
+      </SectionHeading>
       <Flex
         alignItems="center"
         flexDirection="row"
         gap="spacingS"
         marginBottom="spacingM"
       >
-        <Avatar
-          {...args}
-          size="large"
-          colorVariant="primary"
-          icon={<CheckCircleIcon variant="positive" />}
-        />
-        <Avatar
-          {...args}
-          variant="app"
-          size="large"
-          colorVariant="primary"
-          icon={<CheckCircleIcon variant="positive" />}
-        />
+        <Avatar {...args} size="large" colorVariant="primary" />
+        <Avatar {...args} variant="app" size="large" colorVariant="primary" />
         <Avatar {...args} size="large" colorVariant="purple" />
         <Avatar {...args} size="large" colorVariant="yellow" />
         <Avatar {...args} size="large" colorVariant="green" />
         <Avatar {...args} size="large" variant="app" colorVariant="gray" />
         <Avatar {...args} size="large" variant="app" colorVariant="muted" />
-        <Avatar {...args} size="large" colorVariant="pink" variant="app" />
+        <Avatar {...args} size="large" variant="app" colorVariant="pink" />
         <Avatar {...args} size="large" variant="app" colorVariant="lavender" />
         <Avatar {...args} size="large" variant="app" colorVariant="emerald" />
       </Flex>
 
-      <SectionHeading as="h4">Tooltip properties</SectionHeading>
+      <SectionHeading as="h3" marginBottom="spacingS">
+        Tooltip properties
```

Comment:
@andipaetzold Can we force this tooltip open for the storybook?

Reply:
Not sure how to technically achieve that.